src/modules/tests/test_data/test-data.reducer.ts

```diff
@@ -33,6 +33,22 @@ export function testDataReducer(
           [action.payload]: 'DF',
         },
       };
+    case testDataActions.ADD_MANOEUVRE_SERIOUS_FAULT:
+      return {
+        ...state,
+        manoeuvres: {
+          ...state.manoeuvres,
+          [action.payload]: 'S',
```

Comment:
I started us off down this road but could we utilise the `CompetencyOutcome` enum here instead of these strings?

Reply:
Sure, good idea!